src/screens/Routing/RoutingStack.res

```diff
@@ -114,38 +88,37 @@ let make = (~remainingPath, ~previewOnly=false) => {
     None
   }, (pathVar, url.search))
 
-  let getTabName = index => index == 0 ? "active" : "history"
-
   <PageLoaderWrapper screenState>
     <div className={`${widthClass} ${marginClass} gap-2.5`}>
-      <div className="flex flex-col gap-6">
+      <div className="flex flex-col ">
         <PageUtils.PageHeading
-          title="Smart routing configuration"
-          subTitle="Smart routing stack helps you to increase success rates and reduce costs by optimising your payment traffic across the various processors in the most customised yet reliable way. Set it up based on the preferred level of control"
-        />
-        <ActiveRouting.LevelWiseRoutingSection
-          types=[VOLUME_SPLIT, ADVANCED, DEFAULTFALLBACK] onRedirectBaseUrl="routing"
+          title="Smart Routing"
+          subTitle="Smart Routing optimizes payment traffic across processors to boost success rates and reduce costs, offering flexible and reliable control"
         />
+        <RenderIf condition={!previewOnly}>
+          <ActiveRouting routingType />
+        </RenderIf>
+        <div className="mt-6">
+          <div className="font-bold text-black my-2 text-lg">
+            {"Smart Routing Configurations"->React.string}
```

Comment:
@Muditbhatia12  The text-size and text-color of all headings does not match, can you please fix that?